config/environments/production.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  #Tighten the cookies on prod but not in lower envs
+  config.session_store :cookie_store, same_site: :strict
```

Comment:
@ondrae i'm not 100% how initializers differ from `environments` in this case, so if it does have an impact we definitely do want this running on `init` in prod

Reply:
This should work.